src/rancilio-pid.cpp

```diff
@@ -989,6 +990,10 @@ void assignMQTTParam(char *param, double value) {
                         *(uint8_t *)m.mqttVarPtr = value;
                         paramValid = true;
                         break;
+                    case tFloat:
```

Comment:
why add extra float for this one variable? just make it double as well?

Reply:
good idea. I will rework the code to make it double. Solves the problem in the if statement later on

Reply:
It needed to be float as the hx711 lib needs a float as calibration value

Reply:
you can just cast a float to double...

Reply:
But can I cast a double back to float?

Reply:
You can cast in both directions as you like, yes. From double to float, precision loss will occur but doesn't matter here